### PR TITLE
merge master into sj/main

### PR DESCRIPTION
**Pull request by taylorjacklespriggs**

### CODE REVIEW

**Review by dwanderson-intel — APPROVED** [review]
Already ahead of comments I left on other PRs. Awesome

File: sigopt/interface.py [hunk 1]
```diff
@@ -161,6 +160,7 @@ def __init__(self, requestor, api_url=None, user_agent=None, verify_ssl_certs=No
         suggestions,
         experiment_tokens,
         experiment_training_runs,
+        best_training_runs,
```
Comment (dwanderson-intel) [inline sigopt/interface.py:163]: This got doubled-up (it's also second in this list)

**Review by dwanderson-intel — CHANGES_REQUESTED** [review]
Doubled-up resource method

File: sigopt/interface.py (hunk 1 above)
Comment (taylorjacklespriggs) [reply sigopt/interface.py]: fixed! protip in vim: visual select the list content, then `!sort | uniq` to replace with sorted+unique version

Comment (dwanderson-intel) [reply sigopt/interface.py]: ohh, very cool!

**Review by dwanderson-intel — APPROVED** [review]

**Pull request merged**